src/app/components/utility/input/input.component.html

```diff
@@ -15,14 +15,23 @@
     <div class="input-message" #textValue [class.hidden]="toggleErrorMessage()">{{message}}</div>
   </div>
 
+  <div *ngIf="type=='datetime'">
+    <input class="input-field" [(ngModel)]="value" [min]="mindatetime" [owlDateTime]="datetime" [owlDateTimeTrigger]="datetime" (input)="validateInput($event.target.value)" [class.isValid]="isValid" [class.theme-dark]="theme=='dark'" type="{{type}}" [value]="value" [readonly]="isReadonly">
```

Comment:
There is [(ngModel)]="value" and [value]="value" as well, why ?

Reply:
@Shekharrajak cause I want to set the value of this input if `date-picker` value has changed, otherwise I'm unable to get the latest updated value

Reply:
`[(ngModel)]="value" `will bind the latest value. So no need of [value]="value"